addons/jr/Cfgweapons.hpp

```diff
@@ -971,4 +971,30 @@ class CfgWeapons {
             };
         };
     };
+
+    // Contact DLC dummies
+    class acc_flashlight;
+    class acc_flashlight_broken: acc_flashlight {
+        scope = 1; // Optional protected class inheriting from a public base class.
```

Comment:
aren't we hiding the vanilla stuff? or is it already hidden.

Also you renamed file from cfgweapons -> Cfgweapons. But shouldn't it be CfgWeapons?

Reply:
>aren't we hiding the vanilla stuff? or is it already hidden.

These classes are all private in vanilla. But if I inherit when they don't exist, they would get public from their base classes.

>Also you renamed file from cfgweapons -> Cfgweapons. But shouldn't it be CfgWeapons?

Fixed on master